### PR TITLE
Add formula to compute electric charge

### PR DESCRIPTION
**Pull request by Pentiva**
Electric charge (Coulomb) is defined as C = A * s (Current times seconds)

### CODE REVIEW

File: UnitsNet/CustomCode/Quantities/Duration.extra.cs [hunk 1]
```diff
@@ -97,5 +97,11 @@ public TimeSpan ToTimeSpan()
         {
             return Volume.FromCubicMeters(volumeFlow.CubicMetersPerSecond * duration.Seconds);
         }
+
+        /// <summary>Calculate <see cref="ElectricCharge"/> from <see cref="Duration"/> multiplied by <see cref="ElectricCurrent"/>.</summary>
+        public static ElectricCharge operator *(Duration time, ElectricCurrent current)
```
Comment (pgrawehr) [inline UnitsNet/CustomCode/Quantities/Duration.extra.cs:102]: For operator*, both orders should normally be provided. Add `operator *(ElectricCurrent, Duration)` as well.

Comment (Pentiva) [reply UnitsNet/CustomCode/Quantities/Duration.extra.cs:102]: It is provided in ElectricCurrent.extra.cs, as the wiki says the methods should live in the same class as the first parameter.

Comment (pgrawehr) [reply UnitsNet/CustomCode/Quantities/Duration.extra.cs:102]: My bad, sorry.

**Comment by angularsen** [comment]
Nuget on the way.
[Release UnitsNet/4.66.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.66.0)

**Pull request merged**